Userland/Applications/FileManager/main.cpp

```diff
@@ -1313,9 +1307,8 @@ ErrorOr<int> run_in_windowed_mode(DeprecatedString const& initial_location, Depr
 
     paste_action->set_enabled(GUI::Clipboard::the().fetch_mime_type() == "text/uri-list" && access(initial_location.characters(), W_OK) == 0);
 
-    window->set_rect({ left, top, width, height });
-    if (was_maximized)
-        window->set_maximized(true);
+    window->restore_size_and_position("FileManager"sv, { { 640, 480 } });
```

Comment:
This should pass in the default origin that was specified here:
```c++
auto left = Config::read_i32("FileManager"sv, "Window"sv, "Left"sv, 150);
auto top = Config::read_i32("FileManager"sv, "Window"sv, "Top"sv, 75);
```

(150, 75)

Reply:
I disagree a bit because I think that the hardcoded init position is wrong. Let the windowserver decide where the window starts. If there are any tests that rely on this position may we need to add an argument where you can specify a direct position.